MINT.postman_collection.json

```diff
@@ -1,8 +1,8 @@
 {
 	"info": {
-		"_postman_id": "2768f499-6418-47c7-afbd-bc6df44baa68",
+		"_postman_id": "dab8a942-5d1c-47cd-a8f6-f052b439c980",
 		"name": "MINT",
-		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json",
+		"schema": "https://schema.getpostman.com/json/collection/v2.0.0/collection.json",
```

Comment:
It looks like downgrading from 2.1.0 to 2.0.0 caused a bunch of changes in the file -- can you try re-exporting as 2.1.0 and see if these diffs go away?

Reply:
Absolutely. Postman has been giving me some weird issues where it's failing to export. For whatever reason, a PC reboot fixed the issue.